neuro_flow/cli/batch.py

```diff
@@ -30,12 +31,18 @@
 @click.option(
     "--param", type=(str, str), multiple=True, help="Set params of the batch config"
 )
+@click.option(
+    "--meta-from-file",
```

Comment:
Looks at this line, I return to the idea that this name is **too** broad. Until it isn't too late, maybe we have a better idea? My only idea is a `--bake-config-file`, but it isn't perfect too.